osf/migrations/0144_pagecounter_index.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-12 17:10
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False  # CREATE INDEX CONCURRENTLY cannot be run in a txn
+
+    dependencies = [
+        ('osf', '0143_pagecounter_schema'),
+    ]
+
+    operations = [
+        migrations.RunSQL([
+            'CREATE INDEX CONCURRENTLY page_counter_idx ON osf_pagecounter (action, guid_id, file_id);',
+        ], [
+            'DROP INDEX IF EXISTS page_counter_idx, RESTRICT;'
+        ])
```

Comment:
Creates a multi-column index on action, guid, and file fields.  I chose these three instead of adding the version field to it, because this query is more common.   I can add an additional index on all four together if needed.

Reply:
Thinking about it more, I've removed the individual indices off of those fields and added the `version` field to the index.